index.js

```diff
@@ -142,7 +199,17 @@ function getAllMoviesReleasedAtOrBeforeYear() {}
  *  getBiggestBoxOfficeMovie(movies);
  *  //> "Incredibles 2"
  */
-function getBiggestBoxOfficeMovie() {}
+function getBiggestBoxOfficeMovie(movies) {
+  let biggestMovie = ``
+  let movieNumber = 0
+  for (let i=0; i < movies.length; i++){
+    if ( movies[i].boxOffice > movieNumber ){
```

Comment:
I think the issue here is that `boxOffice` is not a number. Instead it is a string that we can use to parse a number from.